playbooks/streisand.yml

```diff
@@ -20,7 +20,8 @@
       when: streisand_openvpn_enabled
     - role: shadowsocks
       when: streisand_shadowsocks_enabled
-    - ssh
+    - role: ssh-forward
+      when: streisand_ssh_forward_enabled
     - role: tinyproxy
       when: streisand_tinyproxy_enabled
```

Comment:
Ahh! I didn't realize it was. Good catch. I'll also update the sanity check role.